query-service-impl/src/main/java/org/hypertrace/core/query/service/pinot/converters/StringToValueConverter.java

```diff
@@ -0,0 +1,38 @@
+package org.hypertrace.core.query.service.pinot.converters;
+
+import com.google.protobuf.ByteString;
+import org.apache.commons.codec.binary.Hex;
+import org.apache.commons.lang.StringUtils;
+import org.hypertrace.core.query.service.api.Value;
+import org.hypertrace.core.query.service.api.ValueType;
+
+/**
+ * Converter to convert any string value to requested {@link Value} of {@link ValueType}
+ * */
+public class StringToValueConverter implements ToValueConverter<String> {
+
+  public static final StringToValueConverter INSTANCE = new StringToValueConverter();
+  private static final String EMPTY = "";
+
+  private StringToValueConverter() {
+  }
+
+  public Value convert(String value, ValueType valueType) throws Exception {
+    Value.Builder valueBuilder = Value.newBuilder();
+    switch (valueType) {
+      case BYTES:
+        String outValue = (StringUtils.isEmpty(value) ||
+            value.equals("null") ||
```

Comment:
Is this correct? `null` is not equal to `''`

Reply:
Should be about right in this case since there shouldn't be any span ids or trace ids which are `bytes[0]` right? any bytes field for that matter.

Reply:
Yes. this is fine as we are converting default string values to a default value of BYTES in pinot which `byte array of length 0`.

Reply:
ok.